prepare/cards/wikitq.py

```diff
@@ -18,7 +20,13 @@
         AddFields({"context_type": "table"}),
         TruncateTableCells(max_length=15, table="table", text_output="answer"),
         TruncateTableRows(field="table", rows_to_keep=50),
-        SerializeTableAsIndexedRowMajor(field_to_field=[["table", "context"]]),
+        PlugInOperator(
+            field="table_serializer",
```

Comment:
I would call it "CustomizableOperator" because it's more directly describes what you can do with it.

Then I would change "field" to "overide_operator_field" so it will be more clear what this field means.

Reply:
Note that people can replace the default operator by any other operator, so it can dramatically change the semantic of the card.